luigi/contrib/s3.py

```diff
@@ -163,236 +181,157 @@ def remove(self, path, recursive=True):
             return False
 
         (bucket, key) = self._path_to_bucket_and_key(path)
-
+        s3_bucket = self.s3.Bucket(bucket)
         # root
         if self._is_root(key):
-            raise InvalidDeleteException('Cannot delete root of bucket at path %s' % path)
-
-        # grab and validate the bucket
-        s3_bucket = self.s3.get_bucket(bucket, validate=True)
+            raise InvalidDeleteException(
+                'Cannot delete root of bucket at path %s' % path)
```

Comment:
Reason for line break here? Is it over 80?

Reply:
Yes it would be 90 without the line break.

Reply:
Oops I just saw you're allowing lines up to 160. Getting rid of the line break now.